packages/kbn-cli-dev-mode/src/config/http_config.ts

```diff
@@ -22,6 +23,7 @@ export const httpConfigSchema = schema.object(
     maxPayload: schema.byteSize({
       defaultValue: '1048576b',
     }),
+    gracefulShutdownTimeout: schema.duration({ defaultValue: '30s' }),
```

Comment:
should the default in Dev mode be 5s because of https://github.com/elastic/kibana/blob/master/packages/kbn-cli-dev-mode/src/cli_dev_mode.ts#L130?

Reply:
I picked 5 seconds arbitrarily, I think we should set the CliDevMode timeout to match core

Reply:
Thank you all! I updated the value of `GRACEFUL_TIMEOUT` to match this value.